controllers/che/checluster_controller.go

```diff
@@ -99,6 +99,10 @@ func NewReconciler(
 	reconcileManager.RegisterReconciler(openshiftoauth.NewOpenShiftOAuth(openShiftOAuthUser))
 	reconcileManager.RegisterReconciler(tls.NewCertificatesReconciler())
 	reconcileManager.RegisterReconciler(tls.NewTlsSecretReconciler())
+	reconcileManager.RegisterReconciler(devworkspace.NewDevWorkspaceReconciler())
+	reconcileManager.RegisterReconciler(rbac.NewRbacReconciler())
```

Comment:
Aren't `GatewayPermissionsReconciler` and `WorkspacePermissionsReconciler` using RBAC? We should be more clear in names.

Reply:
rbac: role-based access control. So it make sense for me to put in that package.